dashboard/src/components/ServiceInstanceList/ServiceInstanceList.test.tsx

```diff
@@ -1,27 +1,117 @@
 import { shallow } from "enzyme";
+import context from "jest-plugin-context";
 import * as React from "react";
 
-import { MessageAlert } from "../ErrorAlert";
-import ServiceInstanceList, { IServiceInstanceListProps } from "./ServiceInstanceList";
+import { IServiceBroker } from "shared/ServiceCatalog";
```

Comment:
we have a mix of relative and non-relative imports here, we should make this more consistent. In my experience, the tests failed with relative imports, but it seems it's not failing here... anyway I think we should use relative imports.

Reply:
this was the automatic import, I don't know what logic that follows.